### PR TITLE
Extend the available options

### PR DESCRIPTION
**Pull request by tboerger**

### CODE REVIEW

File: lib/crowbar/client/command/role/list.rb [hunk 1]
```diff
@@ -19,6 +19,11 @@ module Client
     module Command
       module Role
         class List < Base
+          include Mixin::Barclamp
+
```
Comment (rsalevsky) [inline lib/crowbar/client/command/role/list.rb]: Is this empty line purpose?

Comment (tboerger) [inline lib/crowbar/client/command/role/list.rb]: Yes

**Comment by MaximilianMeister** [comment]
:+1:

**Comment by rsalevsky** [comment]
:+1:

**Pull request merged**